lib/ansible/plugins/doc_fragments/docker.py

```diff
@@ -118,15 +118,15 @@ class ModuleDocFragment(object):
      reinstall of it is required."
 '''
 
-    # Additional, more specific stuff for minimal docker-py version >= 2.0.
-    # Note that docker-py >= 2.0 requires Python 2.7 or newer.
+    # Additional, more specific stuff for minimal Docker SDK for Python version >= 2.0.
+    # Note that Docker SDK for Python >= 2.0 requires Python 2.7 or newer.
 
     DOCKER_PY_2_DOCUMENTATION = r'''
 options: {}
 requirements:
   - "Python >= 2.7"
-  - "Please note that the L(docker-py,https://pypi.org/project/docker-py/) Python
```

Comment:
I do prefer the old explanation, which is more concise.

Reply:
I want to make sure to put "Docker SDK for Python" in there somewhere, so that people aren't confused what we're talking about. Do you think 
```
   - "Docker SDK for Python: Please note that the L(docker-py,https://pypi.org/project/docker-py/) Python
     module has been superseded by L(docker,https://pypi.org/project/docker/)
     (see L(here,https://github.com/docker/docker-py/issues/1310) for details).
     This module does I(not) work with docker-py."
```
would be better (i.e. just prepend `Docker SDK for Python:`)?

Reply:
@felixfontein LGTM

Reply:
I've updated it.